en/docs/develop/restapis/applications.yaml

```diff
@@ -0,0 +1,2962 @@
+          example: 10
+        applications:
+          type: array
+          items:
+            $ref: '#/components/schemas/ApplicationListItem'
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/Link'
+
+    ApplicationListItem:
+      type: object
+      properties:
+        id:
+          type: string
+          example: "85e3f4b8-0d22-4181-b1e3-1651f71b88bd"
+        name:
+          type: string
+          example: "user-portal"
+        description:
+          type: string
+          example: Application representing user portal
+        image:
+          type: string
+          example: 'https://example.com/logo/my-logo.png'
+        accessUrl:
+          type: string
+          example: 'https://example.com/app/login'
+        self:
+          type: string
+          example: "/t/wso2.com/api/server/v1/applications/85e3f4b8-0d22-4181-b1e3-1651f71b88bd"
+
+    ApplicationModel:
+      type: object
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          example: "394b8adcce24c64a8a09a0d80abf8c337bd253de"
+          readOnly: true
+        name:
+          type: string
+          example: pickup
+        description:
+          type: string
+          example: This is the configuration for Pickup application.
+        imageUrl:
+          type: string
+          example: 'https://example.com/logo/my-logo.png'
+        accessUrl:
+          type: string
+          example: 'https://example.com/login'
+        claimConfiguration:
+          $ref: '#/components/schemas/ClaimConfiguration'
+        inboundProtocolConfiguration:
+          $ref: '#/components/schemas/InboundProtocols'
+        authenticationSequence:
+          $ref: '#/components/schemas/AuthenticationSequence'
+        advancedConfigurations:
+          $ref: '#/components/schemas/AdvancedApplicationConfiguration'
+        provisioningConfigurations:
+          $ref: '#/components/schemas/ProvisioningConfiguration'
+
+    ApplicationResponseModel:
+      type: object
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          example: "394b8adcce24c64a8a09a0d80abf8c337bd253de"
+          readOnly: true
+        name:
+          type: string
+          example: pickup
+        description:
+          type: string
+          example: This is the configuration for Pickup application.
+        imageUrl:
+          type: string
+          example: 'https://example.com/logo/my-logo.png'
+        accessUrl:
+          type: string
+          example: 'https://example.com/login'
+        claimConfiguration:
+          $ref: '#/components/schemas/ClaimConfiguration'
+        inboundProtocols:
+          $ref: '#/components/schemas/InboundProtocolsListResponse'
+        authenticationSequence:
+          $ref: '#/components/schemas/AuthenticationSequence'
+        advancedConfigurations:
+          $ref: '#/components/schema
```

Comment:
Decides whether the list of authenticated identity providers need to **be** returned...